### DISCUSSION TITLE
Welcome to Obsidian_to_Anki Discussions!

### DISCUSSION THREAD

**Discussion by Pseudonium** [ObsidianToAnki/Obsidian_to_Anki#111] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Pseudonium** [comment]
Lol I guess I can start this.

Hi, I'm the developer of Obsidian_to_Anki, and a first-year undergrad math student. I use Anki to memorise various definitions, but I also use it to memorise longer explanations, which I find helps my understanding. And I use it quite extensively too!

Cards added with Obsidian_to_Anki: 3007

**Comment by iris2jane** [comment]
Hello, I'm Iris😊. Thanks for your excellent work that I can convert obsidian to anki so smoothly. I'm not good at python, so it was quite hard to use the script. I usually memorize my lecture notes and organize it at the same time, so i focus on the development progress of plugin for a while. Still can't believe it come out so fast!

**Comment by problemadder** [comment]
Hey Pseudonium, thanks a lot for sharing!
I am a long time Anki user. I use it for vocabulary, chess puzzles, and knowledge in general.
Its great to have/create the "cards" within the obsidian notes because this way you just have more context at hand and see the big picture while learning.
Another thing i do is to import MyClippings.txt from kindle directly to my Vault (https://github.com/duarteocarmo/my-personal-zen). With your Plugin its possible to generate flashcards from those reading highlights and still have a good overview.
I would love to see and hear from other people for inspiration how to use obsidian for learning.
